modules/custom/core/govcms8_modifiers/config/install/paragraphs.paragraphs_type.parallax_bg_modifier.yml

```diff
@@ -0,0 +1,9 @@
+uuid: f95f4411-a65d-4bd5-949e-356ec7f4a276
+langcode: en
+status: true
+dependencies: {  }
+id: parallax_bg_modifier
+label: 'Parallax Background Modifier'
```

Comment:
Can we make the Parallax Background Modifier use the same media browser as the featured image field?

Reply:
Fixed in e2cb3b8cb13a28334ef135040adb76017c0a2098